### PR TITLE
test: fix misleading fee unit in mempool_limit.py

### PR DESCRIPTION
**Pull request by theStack**
The PR is a follow-up to #22543. The helper `send_large_txs` in its current interface has a fee_rate parameter, implying that it would create a transaction with exactly that rate. Unfortunately, this fee rate is only passed to MiniWallet's `create_self_transfer` method, which can't know that we append several tx outputs after, increasing the tx's vsize and decreasing it's fee rate accordingly.

In our case, the fee rate is off by several orders of magnitude, as the tx's vsize changes changes from 96 to 67552 vbytes (>700x), i.e. the value passed to this function is neither really a fee rate nor an absolute fee, but something in-between, which is very confusing. It was suggested to simply in-line this helper as it's currently only used in this single test (https://github.com/bitcoin/bitcoin/pull/22543#discussion_r701685136, https://github.com/bitcoin/bitcoin/pull/22543#issuecomment-918986896), but I could imagine that this helper may also become useful for other tests and may be moved to a library (e.g. wallet.py) in the future.

Clarify the interface by passing an absolute fee that is deducted in the end (and also verified, via testmempoolaccept) and also describe how we come up with the value passed. On master, the comment says that the fee rate needs to increased "massively"; this word is also removed because the fee rate only needs to be higher for the test to succeed.

### CODE REVIEW

**Comment by Zero-1729** [comment]
Concept ACK

File: test/functional/mempool_limit.py [hunk 1]
```diff
@@ -23,16 +29,19 @@ def set_test_params(self):
         ]]
         self.supports_cli = False
 
-    def send_large_txs(self, node, miniwallet, txouts, fee_rate, tx_batch_size):
+    def send_large_txs(self, node, miniwallet, txouts, fee, tx_batch_size):
         for _ in range(tx_batch_size):
-            tx = miniwallet.create_self_transfer(from_node=node, fee_rate=fee_rate)['tx']
+            tx = miniwallet.create_self_transfer(from_node=node, fee_rate=0, mempool_valid=False)['tx']
```
Comment (michaelfolkson) [inline test/functional/mempool_limit.py:34]: Should this `fee_rate` be replaced by `fee` here too?

**Review by michaelfolkson — COMMENTED** [review]
Concept ACK

> the value passed to this function is neither really a fee rate nor an absolute fee

Definitely seems optimal to exclusively use absolute fee and fee rate rather than having some hard to define third variable

**Review by stratospher — COMMENTED** [review]
ACK 2600db6.

This change clarifies the problem with the fee rates in the test.

**Pull request merged**